R/dimension_from_existing.R

```diff
@@ -1,20 +1,14 @@
-dimension.from.existing <- list("http://purl.org/linked-data/sdmx/2009/dimension#refPeriod",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#age",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#civilStatus",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#currency",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#educationLev",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#freq",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#occupation",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#refArea",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#sex",
-                                "http://purl.org/linked-data/sdmx/2009/dimension#timePeriod")
-names(dimension.from.existing) <- c("refPeriod",
-                                    "age",
-                                    "civilStatus",
-                                    "currency",
-                                    "educationLev",
-                                    "freq",
-                                    "occupation",
-                                    "refArea",
-                                    "sex",
-                                    'timePeriod')
+#' A list of URIs that can be used to fill in the from.existing argument in add.dimension.column().
+#' @export
+dimension.from.existing <- list(
```

Comment:
Could we have all of the constants start with a capital letter and be shorter, e.g. `Dimensions$refPeriod` would be easier to remember and would avoid conflicting with the names of your functions, etc.